src/ark/segmentation/ez_seg/merge_masks.py

```diff
@@ -123,6 +125,12 @@ def merge_masks_single(
     # Set up list to store merged cell labels
     remove_cells_list = [0]
 
+    # Create a dictionary of the bounding boxes for all object labels
+    object_labels_bounding_boxes = get_bounding_boxes(object_labels)
+
+    # Calculate all cell regionprops for filtering, convert to DataFrame
+    cell_props = pd.DataFrame(regionprops_table(cell_labels, properties=('label', 'centroid', 'major_axis_length')))
```

Comment:
Do we need `'major_axis_length'`? I don't see it used anywhere specifically.

Reply:
good catch, that was from previous code